pm_icecon/tests/regression/test_bt.py

```diff
@@ -19,6 +20,7 @@
 from pm_icecon.masks import get_ps_land_mask, get_ps_pole_hole_mask
 
 
+@pytest.mark.skip(reason='new bt_amsr2_regression files needed')
```

Comment:
We also need to remove these skips for the `bt` regression tests before this PR can be merged.